pkg/test/exporter_collector_test.go

```diff
@@ -208,7 +207,7 @@ func testExporterToCollector(address net.Addr, isMultipleRecord bool, t *testing
 			klog.Fatalf("Got error when sending record: %v", err)
 		}
 		export.CloseConnToCollector() // Close exporting process
-		time.Sleep(2 * time.Second)
+		time.Sleep(time.Second)
```

Comment:
Here as well move the collector process start to go routine.
Remove the for loop over message channel from go routine and put it in test main. Once that is done, we can safely close collector process. We do not need sleep.